script/rvm-install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Install our selected Ruby version defined in the .ruby-version file.
+#
+# Requires:
+# - [rvm](https://rvm.io/)
+#
+
+rvm install $RUBY_VERSION
```

Comment:
I think that this variable isn't available because the calling script didn't export it. But since its a one-liner, should we just put it in the setup script instead?

Reply:
I think that's a good idea 👍